src/endpoints/authentication/impl.cpp

```diff
@@ -214,14 +215,16 @@ namespace irods::http::handler
 
 			if (did_except) {
 				irods::http::globals::background_task([fn = __func__, _sess_ptr, _req = std::move(_req)] {
+					const auto state{irods::http::process_stash::insert(std::chrono::steady_clock::now() + std::chrono::seconds(120))};
```

Comment:
Configurable expire?

Spec doesn't say it should/shouldn't expire, but we don't want or memory to run out from garbage requests.

Reply:
Do we expect the "state" value to be read/fetched within some number of milliseconds if the client is behaving?

Reply:
We expect some reasonable time frame for us to redirect the user, for the user to fill in their OIDC login info, and receive the data they send back.

Reply:
In that case, I think it needs to be configurable or the window needs to be increased.

Reply:
a vote for configurable...

and now... a great naming!

Reply:
Should be able to throw in a name no problem. Can always debate on the name later too.

Ideas for names welcome!

Reply:
Gave it a name, can debate it in new comment/issue if wanted.